specification/vmwarevirtustream/resource-manager/Microsoft.VMwareVirtustream/preview/2016-09-01-preview/vmwarevirtustream.json

```diff
@@ -0,0 +1,1304 @@
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "name": "privateCloudName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the private cloud."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/PrivateCloudRequest"
+            }
+          },
+          {
+            "$ref": "#/parameters/apiVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "schema": {
+              "$ref": "#/definitions/PrivateCloudResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "PrivateClouds_Update": {
+            "$ref": "./examples/PrivateClouds_Update.json"
+          }
+        }
+      },
+      "delete": {
+        "operationId": "PrivateClouds_Delete",
+        "x-ms-long-running-operation": true,
+        "tags": [
+          "PrivateClouds"
+        ],
+        "summary": "delete a private cloud",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "name": "privateCloudName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the private cloud."
+          },
+          {
+            "$ref": "#/parameters/apiVersion"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "accepted operation"
+          },
+          "204": {
+            "description": "no content"
+          }
+        },
+        "x-ms-examples": {
+          "PrivateClouds_Delete": {
+            "$ref": "./examples/PrivateClouds_Delete.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.VMwareVirtustream/privateClouds/{privateCloudName}/clusters": {
+      "get": {
+        "operationId": "Clusters_List",
+        "tags": [
+          "Clusters"
+        ],
+        "summary": "list clusters in a private cloud",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "name": "privateCloudName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the private cloud."
+          },
+          {
+            "$ref": "#/parameters/apiVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "successful operation",
```

Comment:
All List* APIs should support paging (even though if for now, there's never more than one page).

Reply:
`Clusters_List` API does not currently support paging and the list of results is not wrapped in `value`. I'll add an issue for the next API version.

Reply:
It MUST be wrapped in a "value" array even if paging is not supported in v1.

Reply:
@ctaggart are you planning on addressing this comment (re wrapping in a "value")?